### PR TITLE
[plugin.video.eafa] 1.1.0

### PR DESCRIPTION
**Pull request by FraserChapman**
### Description

Updated to be PY3 compatible and use script.module.cache - as discussed with @enen92 

See: https://github.com/xbmc/repo-plugins/pull/2477
See: https://github.com/xbmc/repo-scripts/pull/1115

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.
- If you see no activity on your PR after a week (so at least one weekend has passed) then please go to the #kodi-dev freenode IRC channel to reach out to the team

### CODE REVIEW

**Comment by FraserChapman** [comment]
Just waiting on the module PR - Required dependency script.module.cache is not available in current repository

**Comment by enen92** [comment]
@FraserChapman please readjust to correct branch or resubmit. Dependency is in krypton now.

**Comment by FraserChapman** [comment]
@enen92 - Hey - I closed the other PR - this is a PR for krypton.

File: plugin.video.eafa/changelog.txt [hunk 1]
```diff
@@ -1,2 +1,16 @@
+v.1.1.0
```
Comment (enen92) [inline plugin.video.eafa/changelog.txt]: this is no longer used by kodi (only the news tag), so not needed on the submission. Still, you're free to keep it for a global changelog

Comment (FraserChapman) [reply plugin.video.eafa/changelog.txt]: Ah ok, will probably rm in future as it is one less thing to update - figure the commit history is a better log anyway :) Thanks for the heads up.

**Pull request merged**